src/app/pages/server-listing/server-listing.component.ts

```
@@ -361,8 +371,33 @@
-    HSComboBox.autoInit();
+    try {
+      const prelineModule = await import('@preline/combobox');
+      // try to initialize using the default export, which works in dev
+      if (prelineModule.default && typeof prelineModule.default.autoInit === 'function') {
+        prelineModule.default.autoInit();
+      } else {
+        // need to fall back to manual initialization in prod :shrug:
+        document.querySelectorAll('[data-hs-combobox]').forEach(el => {
+          if (prelineModule.default && typeof prelineModule.default === 'function') {
+            try {
+              // @ts-ignore - Attempting to initialize even if types don't match
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)